tests/storage/iso/test_nfs_iso_sr.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import os
+import logging
+from conftest import copy_tools_iso_to_iso_sr, check_iso_mount_and_read_from_vm
+from conftest import remove_iso_from_sr, nfs_iso_device_config
+
+# Requirements:
+# - one XCP-ng host with a NFS share for the SR and a VM
+
+class TestNFSISOSRCreateDestroy:
+    """
+    Tests that do not use fixtures that setup the SR or import VMs,
+    because they precisely need to test SR creation and destruction
+    """
+
+    def test_create_sr_nfs_with_bad_location(self, host):
+        sr = None
+        try:
+            device_config = nfs_iso_device_config
+            nfs_iso_device_config['location'] = nfs_iso_device_config['location'] + '/wrongpath'
+            host.sr_create('iso', "ISO-NFS-SR", device_config, verify=True)
+        except Exception:
```

Comment:
Can we a test a more precise exception or not? And with or without the expected message?

Reply:
As this applies to all SR tests, a new card was created to handle it later.